Scripts/models/logit.py

```diff
@@ -86,13 +86,24 @@ def _calc_dest_util(self, mode: str, impedance: dict) -> numpy.ndarray:
         size = numpy.zeros_like(utility)
         self._add_zone_util(size, b["attraction_size"])
         impedance["attraction_size"] = size
-        if "transform" in b:
-            b_transf = b["transform"]
-            transimp = numpy.zeros_like(utility)
-            self._add_zone_util(transimp, b_transf["attraction"])
-            self._add_impedance(transimp, impedance, b_transf["impedance"])
-            impedance["transform"] = transimp
```

Comment:
Now we suddenly came up with a use case for this, so I will need to modify this branch a little bit to take into account "transform".